client/signup/wpcom-login-form/index.jsx

```diff
@@ -12,46 +28,27 @@ export default class WpcomLoginForm extends Component {
 		this.form.submit();
```

Comment:
I'm wondering what is Testing Library going to do when the form is submitted on mount. A real form would POST to the `action` URL and navigate to a different page.

Reply:
It's not implemented in jsdom which means it'll only show a warning which says exactly that.